### PR TITLE
Shaymaa

### PR DESCRIPTION
**Pull request by shay0909**

### CODE REVIEW

**Comment by mathurinm** [comment]
@shay0909 you have a conflict, check the readme on how to fix it

**Comment by shay0909** [comment]
@mathurinm hello, please approve my request

**Comment by shay0909** [comment]
@mathurinm hello, the conflict has been resolved. please approve the request

File: students.txt [hunk 1]
```diff
@@ -3,8 +3,8 @@ Last name	First name
 Alberge	Julie
 Arya	Aditeya
 Bairagi	Soumyabrata X
-Bakkass	Shaymaa
-Barry	Ibrahim X
+Bakkass	Shaymaa X 
+Barry	Ibrahim
```
[the remark this answers is not in the document]
Comment (shay0909) [reply students.txt]: @mathurinm i added them and resolved the changes again, normally it should be fine now

**Comment by mathurinm** [comment]
Perfect, great !

**Pull request merged**